doc/classes/RenderingDevice.xml

```diff
@@ -1798,7 +1798,7 @@
 			Image uniform. TODO: Difference between texture and image uniform
 		</constant>
 		<constant name="UNIFORM_TYPE_TEXTURE_BUFFER" value="4" enum="UniformType">
-			Texture buffer uniform. TODO: Difference between texture and texture buffe uniformr
+			Texture buffer uniform. TODO: Difference between texture and texture buffe uniform
```

Comment:
```suggestion
			Texture buffer uniform. TODO: Difference between texture and texture buffer uniform
```
I'd argue we shouldn't have todos in the classref though, they won't ever be handled this way.

Reply:
Yeah, I was thinking about this too. Removed these TODOs in the updated commit.

Reply:
> TODO: Difference between texture and texture buffer uniform

Enum definition have some comments about uniform GLSL types, which is probably good enough to indicate the difference in the docs as well:

```
UNIFORM_TYPE_SAMPLER, //for sampling only (sampler GLSL type)
UNIFORM_TYPE_SAMPLER_WITH_TEXTURE, // for sampling only, but includes a texture, (samplerXX GLSL type), first a sampler then a texture
UNIFORM_TYPE_TEXTURE, //only texture, (textureXX GLSL type)
UNIFORM_TYPE_IMAGE, // storage image (imageXX GLSL type), for compute mostly
UNIFORM_TYPE_TEXTURE_BUFFER, // buffer texture (or TBO, textureBuffer type)
UNIFORM_TYPE_SAMPLER_WITH_TEXTURE_BUFFER, // buffer texture with a sampler(or TBO, samplerBuffer type)
UNIFORM_TYPE_IMAGE_BUFFER, //texel buffer, (imageBuffer type), for compute mostly
UNIFORM_TYPE_UNIFORM_BUFFER, //regular uniform buffer (or UBO).
UNIFORM_TYPE_STORAGE_BUFFER, //storage buffer ("buffer" qualifier) like UBO, but supports storage, for compute mostly
UNIFORM_TYPE_INPUT_ATTACHMENT, //used for sub-pass read/write, for mobile mostly
```